subgraphs/hemi-tunnel-withdrawals-subgraph/src/mappings/l-2-bridge.ts

```diff
@@ -0,0 +1,44 @@
+// Note: This file is written in AssemblyScript. It is required like this to run in the Graph nodes.
+// See docs https://www.assemblyscript.org/ - don't let the .ts extension confuse you. It's fairly limited
+// and many js/ts operations are not supported
+import { dataSource, log } from '@graphprotocol/graph-ts'
+
+import { WithdrawalInitiated as WithdrawalInitiatedEvent } from '../../generated/L2Bridge/L2Bridge'
+import { EvmWithdrawal } from '../entities/evmWithdrawal'
+
+const ethereumMainnetId = 1
+const hemiMainnetId = 43111
+const hemiSepoliaId = 743111
+const sepoliaId = 11155111
```

Comment:
Can these be imported from `utils`?

Reply:
yes, probably yes.

Reply:
Fixed here [46816ec](https://github.com/hemilabs/ui-monorepo/pull/922/commits/46816ec339c22b5ce447860a130e374a81ca246d)